__test__/lib/setup.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
we never got to cover this but 
instead of requireing this file at the top of every single file, jest can be configured to load this before running all tests 
https://facebook.github.io/jest/docs/en/configuration.html#setupfiles-array